lib/Navigation/Renderer/Menu.php

```diff
@@ -756,6 +757,13 @@ protected function _renderMenu(
         foreach ($iterator as $page) {
             $depth = $iterator->getDepth();
             $isActive = $page->isActive(true);
+
+
+            //set ulCLass depth wise
+            if(is_array($ulClasses)) {
+                $ulClass = ($ulClasses[$depth] ? $ulClasses[$depth] : $ulClasses['default']);
```

Comment:
@dvesh3 I think we should better use the cull-coalescing-operator `??` here, because it doesn't issue a warning if the array index doesn't exist (and it's shorter ;)). 
`$ulClass = $ulClasses[$depth] ?? $ulClasses['default']`

Reply:
@dvesh3 updated it already, nothing else to do for you ;)

Reply:
you're a lifesaver 😃